src/SFA.DAS.EAS.Web/Helpers/HtmlHelperExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Web.Mvc;
+using SFA.DAS.Authorization.Services;
+
+namespace SFA.DAS.EAS.Web.Helpers
+{
+    public static class HtmlHelperExtensions
+    {
+        public static bool IsAuthorized(this HtmlHelper htmlHelper, string featureType)
+        {
+            var authorisationService = DependencyResolver.Current.GetService<IAuthorizationService>();
+            var isAuthorized = authorisationService.IsAuthorized(featureType);
+
+            return isAuthorized;
+        }
+    }
+}
```

Comment:
Should this and the file above of the same name actually be in the same file?

Reply:
Moved to existing Extensions file.